CMakeLists.txt

```diff
@@ -14,12 +14,12 @@
 
 cmake_minimum_required(VERSION 3.8)
 
-if(WASM) 
+if(WASM)
     execute_process(COMMAND ${CMAKE_SOURCE_DIR}/wasm/install_deps.sh)
     set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/wasm/emsdk/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake")
 endif()
 
-project(mgclient VERSION 1.2.0)
+project(mgclient VERSION 1.4.0)
```

Comment:
This should be 1.3.0 based on 82c8078d3faea904635afb1d7ebdf5f30f8c9963, but unfortunately 6bd4891741c8d629c429fca60478dc99ce505a73 reverted and we didn't noticed.